src/main/java/gift/dao/MemberDAO.java

```diff
@@ -30,7 +40,7 @@ public Member save(Member member) {
         return member;
     }
 
-    private RowMapper<Member> userRowMapper() {
+    private RowMapper<Member> memberRowMapper() {
```

Comment:
이런 식으로 줄여볼 수도 있겠네요.!
```suggestion
private RowMapper<Member> memberRowMapper() {
        return (rs, rowNum) -> new Member(
                rs.getLong("id"),
                rs.getString("email"),
                rs.getString("password")
        );
    }```